cmd/clusterctl/cmd/generate_provider.go

```diff
@@ -95,6 +96,8 @@ func init() {
 	generateProviderCmd.Flags().BoolVar(&gpo.raw, "raw", false,
 		"Generate configuration without variable substitution in a yaml format.")
 
+	generateProviderCmd.Flags().StringVar(&gpo.outputFile, "write-to", "-", "Specify the output file to write the template to, defaults to STDOUT if the flag is not set")
```

Comment:
```suggestion
	generateProviderCmd.Flags().StringVar(&gpo.outputFile, "write-to", "", "Specify the output file to write the template to, defaults to STDOUT if the flag is not set")
```

Reply:
Thank you for catching that, I missed it.